.changeset/witty-papers-tan.md

```diff
@@ -0,0 +1,124 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/runtime-definitions": minor
+---
+---
+"section": feature
+---
+
+Enable Synchronous Child Datastore Creation
+
+## Overview
+
+This feature introduces a new pattern for creating datastores synchronously within the Fluid Framework. It allows for the synchronous creation of a child datastore from an existing datastore, provided that the child datastore is available synchronously via the existing datastore's registry and that the child's factory supports synchronous creation. This method also ensures strong typing for the consumer.
```

Comment:
Something to consider: The notion of "child" could be interpreted differently depending on context. Here you mean specifically how factories / registry is organized. We do not have notion of child otherwise (i.e. one data store "hosting" a child data store / controlling its runtime behaviors)